README.md

```diff
@@ -111,6 +111,7 @@ If you are using the Docker file, the port is `3300` ([1](https://github.com/sta
 ## Deployment
 
 - [Docker deployment](docs/docker_deployment.md)
+- [Linux deployment](docs/linux_deployment.md)
```

Comment:
This is for clearnet only right? I'm assuming you didn't use tor. Do you know how much more different it is if someone wanted to use tor instead?

Reply:
> This is for clearnet only right?

yes

> Do you know how much more different it is if someone wanted to use tor instead?

no, actually didnt use tor before with proxy (i do have tor browser bc its easy to use)

Reply:
ok sounds good, guide is looking great btw :)